src/averageAges.js

```diff
@@ -15,17 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
```

Comment:
let's first assign default value to 'men' variable and if the 'century' is passed we will reassign the its value
use 'if' instead of ternary, now this part of code look messy and is hard to read